go/vt/vttablet/tabletmanager/vreplication/framework_test.go

```diff
@@ -79,13 +80,13 @@ func init() {
 	tabletconntest.SetProtocol("go.vt.vttablet.tabletmanager.vreplication.framework_test", "test")
 
 	binlogplayer.RegisterClientFactory("test", func() binlogplayer.Client { return globalFBC })
-	flag.Set("binlog_player_protocol", "test")
+	pflag.Set("binlog_player_protocol", "test")
 
 	heartbeatRe = regexp.MustCompile(`update _vt.vreplication set time_updated=\d+ where id=\d+`)
 }
 
 func TestMain(m *testing.M) {
-	flag.Parse() // Do not remove this comment, import into google3 depends on it
+	pflag.Parse() // Do not remove this comment, import into google3 depends on it
```

Comment:
i don't fully understand this; do you?

Reply:
Removed the line and tests continue to work. 

googled(!) for google3 and found it is the internal project name for their code repo. I guess this was from the days google was using vitess internally. I had left it in because I didn't want to break any external dependencies.